src/sentry/integrations/vercel/client.py

```diff
@@ -35,9 +35,14 @@ def request(self, method, path, data=None, params=None, allow_text=False):
             params = params or {}
             params["teamId"] = self.team_id
         headers = {"Authorization": u"Bearer {}".format(self.access_token)}
-        return self._request(
-            method, path, headers=headers, data=data, params=params, allow_text=allow_text
-        )
+        try:
+            return self._request(
+                method, path, headers=headers, data=data, params=params, allow_text=allow_text
+            )
+        except ApiError as e:
+            if e.code == 402:
+                raise IntegrationError("Payment Required")
+            raise IntegrationError(e.text)
```

Comment:
I don't know if this is what I should be doing, but I couldn't get good text from the error so I hard coded it for the 402. For the rest ... ?

Reply:
hmm do we want to raise the `IntegrationError` here? or do we re-raise the `ApiError`?